config/opensearch_dashboards.yml

```diff
@@ -305,3 +305,7 @@
 
 # Set the value to true to enable workspace feature
 # workspace.enabled: false
+
+# Set the value to true to enable Ui Metric Collectors in Usage Collector
+# This publishes the Application Usage and UI Metrics into the saved object, which can be accessed by /api/stats?extended=true&legacy=true&exclude_usage=false
+# usageCollection.uiMetric.enabled: false
```

Comment:
Do you have a chance to try this feature flag? It seems that `usageCollection.uiMetric.enabled: false` still writes application_usage to the .kibana index.

Reply:
Thanks @Flyingliuhub for spotting the error. 

This issue is not related to my current change rather can also be found in the current main branch. I have found the root cause of the issue and have included a fix with this PR.

**How to reproduce this issue?**
- Run opensearch dashboards
- Call the /api/stats?extended=true&legacy=true&exclude_usage=false important include the exclude_usage=false flag 

**Root Cause**:
Vega Visualization is not using ui_metric collector but rather is using a custom version of usage_collector for itself. When we call the api/stats with exclude_usage=false flag, It asks all usage collectors to fetch their metrics which would be consolidated and sent back. The fetching metric method for Vega is trying to fetch the stats from an index. It was unable to resolve the index properly from the config.

It was currently trying to find index field within kibana section of config however this information is present within opensearchDashboards section of config. Changing the path has helped to resolve the issue.

Config:
`{
  opensearchDashboards: {
    index: '.kibana'
  }
}` 

**Why was it not noticed till now?**
The workflow to fetch metric from usage collectors can be triggered either by telemetry or by calling the api/stats API with exclude_usage=false flags. Since telemetry was disabled and api/stats was not being called with exclude_usage=false flags this issue wasn't uncovered.

Reply:
@LDrago27 can you create a new issue for this?

Reply:
Sure. https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6376